mycroft/client/speech/mic.py

```diff
@@ -374,14 +364,12 @@ def __init__(self, wake_word_recognizer, watchdog=None):
 
         # The maximum seconds a phrase can be recorded,
         # provided there is noise the entire time
-        self.recording_timeout = listener_config.get('recording_timeout',
-                                                     self.RECORDING_TIMEOUT)
+        self.recording_timeout = listener_config.get('recording_timeout')
```

Comment:
Maybe use brackets here to generate a lookup error if it's missing from config. This will default to `None` and cause issues when used.

Reply:
should i just include the previous hardcoded values as a sane backup default?

Reply:
This is a case where I'm not quite sure what I think, if these are completely missing the default config is either missing or messed up. Sane defaults are generally a good thing though and the user would get a warning that the config failed to load if that were the case. So you're probably right.